marc_to_solr/lib/electronic_access_link.rb

```diff
@@ -44,6 +44,10 @@ def url
       if !@url_key&.match?(URI.regexp)
         @logger.error "#{@bib_id} - invalid URL for 856$u value: #{@url_key}"
         @url_key = nil
+      elsif @url_key.start_with?(/http:\/[A-Za-z]/)
```

Comment:
This is needed because Ruby's built-in `URI.regexp` does not detect that "http:/whatever" is an invalid URL.